packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -877,9 +878,15 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
       });
     };
 
-    updateQueryFromURL(urlState.s);
-    updateSortFromURL(urlState.sort);
-  }, [urlState, searchQueryParser, getTagList, urlStateEnabled]);
+    if (urlStateEnabled) {
+      updateQueryFilter(urlState.s);
+      updateSortFromURL(urlState.sort);
+    }
+
+    if (initialQuery) {
```

Comment:
This is to align initialFilter and initial selected tags as well. Otherwise the searchQuery is incorrect as we use Ast: [] to init for the initial query.